server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueue.java

```diff
@@ -119,4 +121,11 @@ public String toString() {
             .add("name", name)
             .toString();
     }
+
+    public void rebuild(Instant olderThan) {
```

Comment:
rebuild is also here not good enought. How about `republishOldReferencesToRabbitMQ` ?

Reply:
because we already are in `RabbitMQMailQueue`